### ISSUE TITLE
Does not support local repository resolution

### ISSUE THREAD

**Issue by pfn** [alexarchambault/coursier#165]
```
> update
Warning: unrecognized Maven repository protocol in file:/C:/Users/pfnguyen/android-sdk-windows/extras/google/m2repository/, ignoring it
Warning: unrecognized Maven repository protocol in file:/C:/Users/pfnguyen/android-sdk-windows/extras/android/m2repository/, ignoring it
```

**Comment by alexarchambault** [comment]
It's fixed in the current master (https://github.com/alexarchambault/coursier/pull/155). It'll be in the next milestone, in a few days.

**Comment by alexarchambault** [comment]
Fixed since https://github.com/alexarchambault/coursier/pull/155, included in `1.0.0-M9`. Feel free to re-open if you still face problems.
